man/systemd-escape.xml

```diff
@@ -135,6 +137,19 @@
         <option>--unescape</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--instance</option></term>
+
+        <listitem><para>With <option>--unescape</option>, unescape
+        and print only the instance part of an instantiated unit name
+        template. Results in the empty string for an uninstantiated
+        template like <filename>ssh@.service</filename> and an error
```

Comment:
I think it's better to error out with a template name, because that it more likely to be an error than intentional...

Reply:
Makes sense, since you can’t use a unit with an empty template name.